assets/shaders/show_prepass.wgsl

```diff
@@ -23,6 +24,9 @@ fn fragment(
     } else if settings.show_normals == 1u {
         let normal = prepass_normal(frag_coord, sample_index);
         return vec4(normal, 1.0);
+    } else if settings.show_motion_vectors == 1u {
+        let motion_vector = prepass_motion_vector(frag_coord, sample_index);
```

Comment:
```suggestion
        let motion_vector = prepass_motion_vector(frag_coord, sample_index) / globals.delta_time;
```
add delta time so that high refresh rates still show some visible motion. 
could also add +0.5 so that negatives are discernable, but it kind of looks nicer without.